### ISSUE TITLE
https://chan.sankakucomplex.com/ comes only to page 50

### ISSUE THREAD

**Issue by devilwar93** [Bionus/imgbrd-grabber#505]
hello , I do know that the product has flaws https://chan.sankakucomplex.com/ again , just let me come now to page 50 , before I came to the 100
![captura 1](https://cloud.githubusercontent.com/assets/17387903/13999269/131b7474-f101-11e5-8cd9-4c2863316413.JPG)

**Comment by Bionus** [comment]
Could it be that you just hit their throttle limit? If you make too many requests and too fast they block you for three minutes.
What does the `main.log` file says? (located in "%appdata%/Bionus/Grabber")

**Comment by Rika-chan** [comment]
Using those settings the same behavior is observed. I do not end up getting blocked for three minutes either way, however.

The line from main.log when attempting to load a page 51:
```
[19:00:19.460] Receiving page https://chan.sankakucomplex.com/post/error, https://chan.sankakucomplex.com/post/error, https://chan.sankakucomplex.com/post/error
```

**Comment by Bionus** [comment]
Indeed, my bad.
See issue #523, it should be fixed in the upcoming 4.5.0 version. :smile:

Don't forget to login though, as Sankaku now don't allow non-logged in users to go further than page 50.
